lgsm/functions/install_mta_resources.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# LGSM install_mta_resources.sh function
+# Author: Daniel Gibbs
+# Contributor: ChaosMTA
+# Website: https://gameservermanagers.com
+# Description: Installs the libmysqlclient for database functions on the server and optionally installs default resources required to run the server
+
+local commandname="INSTALL"
+local commandaction="Install"
+local function_selfname="$(basename $(readlink -f "${BASH_SOURCE[0]}"))"
+
+fn_install_libmysqlclient16(){
+	echo ""
+	echo "Checking if libmysqlclient16 is installed"
+	echo "================================="
+	sleep 1
+	if [ ! -f /usr/lib/libmysqlclient.so.16 ]; then
+		fn_print_warn_nl "libmysqlclient16 not installed. Installing.."
+		sleep 1
+		sudo -v > /dev/null 2>&1
+		if [ $? -eq 0 ]; then
+    	fileurl="https://nightly.mtasa.com/files/modules/64/libmysqlclient.so.16"; filedir="${tmpdir}"; filename="libmysqlclient.so.16"; executecmd="executecmd" run="norun"; force="noforce"; md5="6c188e0f8fb5d7a29f4bc413b9fed6c2"
+    	fn_fetch_file "${fileurl}" "${filedir}" "${filename}" "${executecmd}" "${run}" "${force}" "${md5}"
+			sudo mv ${tmpdir}/${filename} /usr/lib/${filename}
```

Comment:
I can't change this. There is no package in apt or yum that will satisfy this requirement. It has to be installed this way. I can make it be a yes/no prompt if you'd prefer or remove it all together. But either way this needs to be done in order for the server to use database functions.